modules/ROOT/partials/integrations/react-bundling.adoc

```diff
@@ -41,9 +41,9 @@ import 'tinymce/skins/ui/oxide/content.js';
 export default function TinyEditorComponent(props) {
   return (
     <Editor
-      licenseKey='your-license-key'
+      licenseKey='gpl' // gpl for open source, T8LK:... for commercial
```

Comment:
* [ ] Shift the comment to above `return`

Reply:
```suggestion
      licenseKey='gpl'
```